Gems/Maestro/Code/Source/Components/SequenceComponent.cpp

```diff
@@ -99,6 +99,12 @@ namespace Maestro
 
     SequenceComponent::SequenceComponent()
     {
+        AZ_Info("SequenceComponent", "SequenceComponent");
```

Comment:
maybe put the sequence name if it exist or the entity it belongs to in this log (not sure why we want it)